packaging/ubuntu-16.04/changelog

```diff
@@ -1,3 +1,27 @@
+snapd (2.58.3) xenial; urgency=medium
+
+  * New upstream release, LP: #1998462
+    - interfaces/screen-inhibit-control: Add support for xfce-power-
```

Comment:
not something to fix here, but this PR didn't follow the proper summary conventions, Add/add